package.json

```diff
@@ -65,9 +71,16 @@
     "npm-run-all": "4.1.5",
     "prettier": "1.18.2",
     "prop-types": "15.7.2",
+    "react": "16.9.0",
+    "react-async": "^8.0.0-alpha.0",
+    "react-dom": "16.9.0",
+    "storybook-chromatic": "2.2.2"
+  },
+  "resolutions": {
+    "typescript": "3.6.3",
     "react": "16.10.2",
     "react-async": "8.0.0",
     "react-dom": "16.10.2",
     "storybook-chromatic": "3.0.3"
   }
```

Comment:
This was an error on merge.
Just a quick question, since I noticed it here: is the requirement of `"react-async": "8.0.0"` on purpose?

Reply:
OK I had to remove `"react-async": "8.0.0"` as a dependecy since it caused errors in chromatic. (It ran with the old version, which didn't have `ActionTypes` but `actionTypes`)